filecoindot_proofs/src/verify.rs

```diff
@@ -44,3 +45,10 @@ impl ProofVerify {
         Self::traverse_and_match::<N>(proof, index - 1, target_cid)
     }
 }
+
+pub(crate) type NodeType =
```

Comment:
Note: `NodeType` is copied from `forest_hamt_adaptor` test.

Reply:
@willeslau is it a good idea to make this`NodeType` be in the filecoindot_proofs? This is needed in order to expose a `generic_verify` method.